large number of tests fail in CI due to new NewtonSoft.Json version

Example error:

```
WorkspaceServer.CommandLineInvocationException : Microsoft (R) Build Engine version 16.3.0+0f4c62fea for .NET Core\r\nCopyright (C) Microsoft Corporation. All rights reserved.\r\n\r\nC:\\Users\\VssAdministrator\\AppData\\Local\\Temp\\qylmzbwr.raq\\MLS.Blazor\\MLS.Blazor.csproj : error NU1605: Detected package downgrade: Newtonsoft.Json from 12.0.3 to 12.0.2. Reference the package directly from the project to select a different version. \r\nC:\\Users\\VssAdministrator\\AppData\\Local\\Temp\\qylmzbwr.raq\\MLS.Blazor\\MLS.Blazor.csproj : error NU1605: MLS.Blazor -> blazor-console -> Newtonsoft.Json (>= 12.0.3) \r\nC:\\Users\\VssAdministrator\\AppData\\Local\\Temp\\qylmzbwr.raq\\MLS.Blazor\\MLS.Blazor.csproj : error NU1605: MLS.Blazor -> Newtonsoft.Json (>= 12.0.2)\r\n Restore completed in 163.44 ms for C:\\Users\\VssAdministrator\\AppData\\Local\\Temp\\qylmzbwr.raq\\projects\\build\\blazor-console.csproj.\r\n Restore failed in 167.54 ms for C:\\Users\\VssAdministrator\\AppData\\Local\\Temp\\qylmzbwr.raq\\MLS.Blazor\\MLS.Blazor.csproj.\r\n\r\nExit code 1:
```

--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -17,7 +17,7 @@
 
   <PropertyGroup>
    <!-- These properties are needed so that version number can be same for MLS.Agent.Tools and the libraries that depend on it -->  
-    <NewtonsoftJsonVersion>12.0.02</NewtonsoftJsonVersion>
+    <NewtonsoftJsonVersion>12.0.3</NewtonsoftJsonVersion>
     <PocketLoggerVersion>0.3.0</PocketLoggerVersion>
     <SystemDiagnosticsProcessVersion>4.3.0</SystemDiagnosticsProcessVersion>
     <SystemReactiveVersion>4.1.6</SystemReactiveVersion>

--- a/MLS.Agent.Tests/MLS.Agent.Tests.csproj
+++ b/MLS.Agent.Tests/MLS.Agent.Tests.csproj
@@ -27,7 +27,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="3.0.0-rc1.19457.4" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.02" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="Pocket.Disposable" Version="1.0.5">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>

--- a/MLS.Blazor/MLS.Blazor.csproj
+++ b/MLS.Blazor/MLS.Blazor.csproj
@@ -26,7 +26,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="3.0.0-preview9.19457.4" />
     <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="3.0.0-preview9.19424.4" PrivateAssets="all" />
     <PackageReference Include="Microsoft.AspNetCore.Blazor.DevServer" Version="3.0.0-preview9.19424.4" PrivateAssets="all" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
   </ItemGroup>
 
   <ItemGroup>

--- a/Microsoft.DotNet.Interactive.Jupyter/Microsoft.DotNet.Interactive.Jupyter.csproj
+++ b/Microsoft.DotNet.Interactive.Jupyter/Microsoft.DotNet.Interactive.Jupyter.csproj
@@ -9,7 +9,7 @@
     <PackageReference Include="Markdig" Version="0.17.1" />
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="3.0.0" />
     <PackageReference Include="NetMQ" Version="4.0.0.207" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.02" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="JsonSerializationExtensions" Version="0.1.8580001">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>

--- a/Microsoft.DotNet.Interactive.Rendering/Microsoft.DotNet.Interactive.Rendering.csproj
+++ b/Microsoft.DotNet.Interactive.Rendering/Microsoft.DotNet.Interactive.Rendering.csproj
@@ -15,7 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.02" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="pocket.disposable" Version="1.0.5">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>

--- a/Microsoft.DotNet.Try.Project/Microsoft.DotNet.Try.Project.csproj
+++ b/Microsoft.DotNet.Try.Project/Microsoft.DotNet.Try.Project.csproj
@@ -14,7 +14,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="System.ComponentModel.Annotations" Version="4.6.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.02" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
   </ItemGroup>
 
   <ItemGroup>

--- a/Microsoft.DotNet.Try.Protocol.ClientApi/Microsoft.DotNet.Try.Protocol.ClientApi.csproj
+++ b/Microsoft.DotNet.Try.Protocol.ClientApi/Microsoft.DotNet.Try.Protocol.ClientApi.csproj
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.02" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="Pocket.Disposable" Version="1.0.5">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>

--- a/Microsoft.DotNet.Try.Protocol/Microsoft.DotNet.Try.Protocol.csproj
+++ b/Microsoft.DotNet.Try.Protocol/Microsoft.DotNet.Try.Protocol.csproj
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.02" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="Pocket.Disposable" Version="1.0.5">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>

--- a/WasmCodeRunner/MLS.WasmCodeRunner.csproj
+++ b/WasmCodeRunner/MLS.WasmCodeRunner.csproj
@@ -10,7 +10,7 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.02" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="system.commandline.experimental" Version="0.3.0-alpha.19508.1" />
   </ItemGroup>
 </Project>

--- a/WorkspaceServer/Packaging/BlazorPackageInitializer.cs
+++ b/WorkspaceServer/Packaging/BlazorPackageInitializer.cs
@@ -24,7 +24,7 @@ namespace WorkspaceServer.Packaging
 
             var requiredPackages = new List<(string packageName, string packageVersion, string restoreSources)>
             {
-                ("Newtonsoft.Json", "12.0.02", ""),
+                ("Newtonsoft.Json", "12.0.3", ""),
                 ("system.commandline.experimental", "0.3.0-alpha.19317.1", "")
             };
 

--- a/WorkspaceServer/WorkspaceServer.csproj
+++ b/WorkspaceServer/WorkspaceServer.csproj
@@ -55,7 +55,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.Scripting.Common" Version="3.4.0-beta2-19467-02" />
     <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="3.4.0-beta2-19467-02" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.0.0" />
-    <PackageReference Include="NewtonSoft.Json" Version="12.0.2" />
+    <PackageReference Include="NewtonSoft.Json" Version="12.0.3" />
     <PackageReference Include="system.commandline.experimental" Version="0.3.0-alpha.19508.1" />
     <PackageReference Include="Pocket.Disposable" Version="1.0.5">
       <PrivateAssets>all</PrivateAssets>

